app/routes/search.tsx

```diff
@@ -1,4 +1,4 @@
-import * as Popover from '@radix-ui/react-popover'
+import Popover from '@radix-ui/react-popover/dist/index.js'
```

Comment:
Let's open an issue on radix to make this better.

Reply:
there are more then one issue about it
https://github.com/radix-ui/primitives/issues/1835

it does not look like they do something about it

Reply:
I've pushed a PR to [radix-ui](https://github.com/radix-ui/primitives/pull/2130).